resources/views/filament/resources/achievement-resource/pages/logic.blade.php

```diff
@@ -0,0 +1,259 @@
+            $triggerDecoderService->addCodeNotes($groups, $record->game_id);
+
+            $hasAddAddress = $triggerViewerService->hasAddAddressFlag($groups);
+            $addrFormat = $triggerViewerService->getAddressFormat($groups);
+            $markdownOutput = $triggerViewerService->generateMarkdown($groups);
+
+            $tooltipStyle = 'white-space: pre-wrap; font-family: ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, monospace; font-size: 11px';
+        @endphp
+
+        <div
+            x-data="{
+                showDecimal: true,
+                showAliases: false,
+                collapseAddAddress: false,
+                copyMarkdown() {
+                    navigator.clipboard.writeText(@js($markdownOutput));
+                    new FilamentNotification()
+                        .title('Copied to clipboard')
+                        .success()
+                        .send();
+                }
+            }"
+            class="space-y-6"
+        >
+            {{-- Version History --}}
+            @php
+                $versionData = $this->getVersionHistoryData();
+                $triggers = $versionData['triggers'];
+                $summaries = $versionData['summaries'];
+                $diffs = $versionData['diffs'];
+            @endphp
+            @include('filament.resources.achievement-resource.partials.version-history')
+
+            <span>&nbsp;</span>
+
+            {{-- Toolbar --}}
+            <div class="flex flex-wrap items-center justify-between gap-x-6 gap-y-3 p-4 rounded-xl bg-white shadow-sm dark:bg-white/5 ring-1 ring-gray-950/5 dark:ring-white/10">
+                <div class="flex flex-wrap items-center gap-x-6 gap-y-3">
+                    <label class="inline-flex items-center gap-2 cursor-pointer select-none">
+                        <x-filament::input.checkbox x-model="showDecimal" />
+                        <span class="text-sm text-neutral-600 dark:text-neutral-400">Decimal values</span>
+                    </label>
+
+                    <label class="inline-flex items-center gap-2 cursor-pointer select-none">
+                        <x-filament::input.checkbox x-model="showAliases" />
+                        <span class="text-sm text-neutral-600 dark:text-neutral-400">Show aliases</span>
+                    </label>
+
+                    @if ($hasAddAddress)
+                        <label class="inline-flex items-center gap-2 cursor-pointer select-none">
+                            <x-filament::input.checkbox x-model="collapseAddAddress" />
+                            <span class="text-sm text-neutral-600 dark:text-neutral-400">Collapse AddAddress</span>
+                        </label>
+                    @endif
+                </div>
+
+                <x-filament::button size="sm" color="gray" icon="heroicon-o-clipboard-document" @click="copyMarkdown()">
+                    Copy Markdown
+                </x-filament::button>
```

Comment:
This always copies the original view (decimal, no aliases, no collapsed addresses). Is that intentional?